src/card_operations/card_fetch_share.h

```diff
@@ -48,7 +50,9 @@ typedef struct card_fetch_share_cfg {
  *
  * @param config A pointer to the configuration of the card fetch and share
  * operation.
+ * @param response Pointer to buffer where response will be filled
  * @return A card_error_type_e value representing the result of the operation.
  */
-card_error_type_e card_fetch_share(card_fetch_share_cfg_t *config);
+card_error_type_e card_fetch_share(card_fetch_share_configuration_t *config,
```

Comment:
let's make the first parameter as `const` for sanity

Reply:
Updated